src/amo/pages/UserProfileEdit/index.js

```diff
@@ -399,11 +393,8 @@ export class UserProfileEditBase extends React.Component<Props, State> {
 
   preventSubmit() {
     const { user, isUpdating } = this.props;
-    const { username } = this.state;
 
-    return (
-      !user || isUpdating || !username || (username && username.trim() === '')
-    );
+    return !user || isUpdating;
```

Comment:
is there a test that needs to be updated for this?

Reply:
re-added a test case